spec/controllers/pokemons_controllers_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+require 'devise'
+require 'vcr'
+
+RSpec.describe PokemonsController, type: :controller do
+  before do
+    @request.env["devise.mapping"] = Devise.mappings[:user]
+    user = create(:user)
+    sign_in user
+  end
+
+  describe 'GET #search', :vcr do
+    it 'returns a Pokémon when a valid name is provided' do
```

Comment:
non-blocking, removeria o acento do pokemon

```suggestion
    it 'returns a Pokemon when a valid name is provided' do
```